nuxeo-coldstorage-web/src/main/resources/web/nuxeo.war/ui/coldstorage/nuxeo-move-content-to-coldstorage-button.html

```diff
@@ -77,9 +77,20 @@ <h2>[[i18n('documentContentView.moveToColdStorage.ask.confirmation')]]</h2>
           !this.hasFacet(document, 'ColdStorage')
       && (this._hasAdministrationPermissions(currentUser) || this.hasPermission(document, 'WriteColdStorage'))
       && this.hasContent(document)
+      && this._isRenditionAvailable(document)
         );
       }
 
+      _isRenditionAvailable(document) {
+        let result = true;
+        if (document.type === 'Picture') {
```

Comment:
Any particular reason for not checking instead if the document has facet "Picture" or "Video"?

Reply:
Good point! I forgot about facets. chaning the code, thanks for the feedback!